<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TmpWindow</class>
 <widget class="QMainWindow" name="TmpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#TmpWindow{border-image: url(:/new/prefix1/tmpwindow.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
border-image: url(:/new/prefix1/button_yellow.png);

</string>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
border-image: url(:/new/prefix1/button_grey.png);
</string>
    </property>
    <property name="text">
     <string>Back to Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
border-image: url(:/new/prefix1/button_blue.png);
</string>
    </property>
    <property name="text">
     <string>Save to Saving1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Handwriting</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
border-image: url(:/new/prefix1/button_blue.png);
</string>
    </property>
    <property name="text">
     <string>Save to Saving2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
